### PR TITLE
Support for higher image resolution with VC0706 compatible 2 MP cameras

### PR DESCRIPTION
**Pull request by astuder**
This pull request adds support for higher resolution images with VC0706 compatible 2 megapixel cameras , like for example Spinel SC20MPB:
https://www.spinelelectronics.com/2MP_serial_cameras/SC20MPB

The changes are transparent to lower resolution cameras.

This code was tested with:
Adafruit Feather M0 Express
Spinel SC20MPB 2 MP Serial JPEG Camera

### CODE REVIEW

File: Adafruit_VC0706.cpp [hunk 1]
```diff
@@ -173,13 +173,20 @@ uint8_t Adafruit_VC0706::getImageSize() {
...
     @brief  Set image size with VC0706_WRITE_DATA
-    @param x VC0706_640x480, VC0706_320x240 or VC0706_160x120
+    @param x VC0706_640x480, VC0706_320x240, VC0706_160x120, etc.
     @return True on command success
 */
 /**************************************************************************/
 boolean Adafruit_VC0706::setImageSize(uint8_t x) {
   uint8_t args[] = {0x05, 0x04, 0x01, 0x00, 0x19, x};
 
+  if (x < 0x33)
```
Comment (ladyada) [inline Adafruit_VC0706.cpp:183]: change to `VC0706_1024x768` ?

**Review by ladyada — CHANGES_REQUESTED** [review]

File: Adafruit_VC0706.cpp (hunk 1 above)
Comment (astuder) [reply Adafruit_VC0706.cpp]: good point

File: Adafruit_VC0706.cpp [hunk 2]
```diff
@@ -389,8 +396,10 @@ uint8_t Adafruit_VC0706::getCompression(void) {
 /**************************************************************************/
 boolean Adafruit_VC0706::setPTZ(uint16_t wz, uint16_t hz, uint16_t pan,
                                 uint16_t tilt) {
-  uint8_t args[] = {0x08,     wz >> 8, wz,        hz >> 8, wz,
-                    pan >> 8, pan,     tilt >> 8, tilt};
+  uint8_t args[] = {
+      0x08,         (uint8_t)(wz >> 8),  (uint8_t)wz,  (uint8_t)(hz >> 8),
+      (uint8_t)hz,  (uint8_t)(pan >> 8), (uint8_t)pan, (uint8_t)(tilt >> 8),
```
Comment (ladyada) [inline Adafruit_VC0706.cpp:401]: nice fix!

**Review by ladyada — APPROVED** [review]
untested but looks good, thank you!

File: Adafruit_VC0706.cpp (hunk 2 above)
Comment (astuder) [reply Adafruit_VC0706.cpp:401]: I only noticed while fighting with the clang formatter :)

**Comment by ladyada** [comment]
thanks ! :)

**Pull request merged**